File: backend/src/main/java/com/hyunprise/backend/domain/coupon/vo/CouponSummary.java
package com.hyunprise.backend.domain.coupon.vo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Timestamp;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CouponSummary {
    private String issuedCouponUUID;
    private String couponName;
    private Integer status;
    private Timestamp expirationDate;

    private String couponUUID;
    private Integer expirationDays;
    private Timestamp creationDate;
    private String brandName;

}

File: android/app/src/main/java/com/hyunprise/android/api/coupon/vo/CouponSummary.kt
package com.hyunprise.android.api.coupon.vo

import java.sql.Timestamp

data class CouponSummary (
    val issuedCouponUUID: String,
    val couponName: String,
    var status: Int,
    val expirationDate: Timestamp,

    val couponUUID:String,
    val expirationDays: Int,
    val creationDate: Timestamp,
    val brandName:String
)